apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/start/page.tsx

```diff
@@ -86,6 +86,9 @@ export default async function Checkout({
     redirectSearchParams.postalCode = cart.taxAddress.postalCode;
   }
 
+  redirectSearchParams.cartId = cart.id;
+  redirectSearchParams.cartVersion = cart.version.toString();
```

Comment:
Since the same cart will be used after login, it shouldn't be necessary to include countryCode and postalCode as params anymore. Could those be removed?

Reply:
Oh probably should have mentioned this comment was regarding lines 84-87.